esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -679,9 +679,8 @@ bool CFileSprayEx::onDFUWUSearch(IEspContext &context, IEspDFUWUSearchRequest &
 {
     try
     {
-#ifdef _CONTAINERIZED
```

Comment:
What happens if someone calls this api in the containerized version, or is the service not published?

Reply:
It is published because #ifdef cannot be used in the ecm file. When someone calls it in the containerized version, ESP returns an empty list of ClusterNames.